packages/frontend/src/components/SchedulersView/index.tsx

```diff
@@ -0,0 +1,121 @@
+                    align="center"
+                    pr="md"
+                    spacing="xs"
+                    sx={{
+                        flexGrow: 1,
+                    }}
+                >
+                    <Tabs.List>
+                        <Tabs.Tab
+                            value="scheduled-deliveries"
+                            icon={
+                                <MantineIcon
+                                    icon={IconSend}
+                                    size="md"
+                                    color="gray.7"
+                                />
+                            }
+                        >
+                            <Title order={6} fw={500} color="gray.7">
+                                All schedulers
+                            </Title>
+                        </Tabs.Tab>
+                        <Tabs.Tab
+                            value="run-history"
+                            icon={
+                                <MantineIcon
+                                    icon={IconClock}
+                                    size="md"
+                                    color="gray.7"
+                                />
+                            }
+                        >
+                            <Title order={6} fw={500} color="gray.7">
+                                Run history
+                            </Title>
+                        </Tabs.Tab>
+                    </Tabs.List>
+                    <Tooltip label="Click to refresh the status of the scheduled deliveries">
+                        <ActionIcon
+                            ml="auto"
+                            onClick={() =>
+                                queryClient
+                                    .invalidateQueries('schedulerLogs')
+                                    .then(() =>
+                                        showToastSuccess({
+                                            title: 'Scheduled deliveries refreshed successfully',
+                                        }),
+                                    )
+                            }
+                        >
+                            <MantineIcon
+                                icon={IconRefresh}
+                                size="lg"
+                                color="gray.6"
+                                stroke={2}
+                            />
+                        </ActionIcon>
+                    </Tooltip>
+                </Group>
+                <Tabs.Panel value="scheduled-deliveries">
+                    {data && data.schedulers.length > 0 ? (
+                        <Schedulers {...data} projectUuid={projectUuid} />
+                    ) : (
+                        emptyState
+                    )}
+                </Tabs.Panel>
+                <Tabs.Panel value="run-history">
+                    {data &&
+                    data.schedulers.length > 0 &&
```

Comment:
shouldn't this still render if schedulers are 0?

you also check `data && data.schedulers.length > 0 ?` below which is confusing in terms of how logic branches

Reply:
if there are no schedulers, there can't be logs so no

Reply:
I will fix the statement though